src/install.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Install
+ * A simple FIFO queue for installing plugins
+ */
+
+export default function install ([plugin, ...tail], app, callback) {
+  if (!plugin) return callback()
+
+  plugin.register(app, app._options, function(err) {
+    if (err) throw err
+    install(tail, app, callback)
```

Comment:
Basically, run a plugin, when it finishes check if any others exist, then run those. This works sort of like `.then` with promises, but Promises are terrible about hiding errors and I'd like to avoid them where possible.